include/FreeRTOS_IP.h

```diff
@@ -350,6 +361,11 @@
  * be defined in a user module. */
     extern BaseType_t xApplicationGetRandomNumber( uint32_t * pulNumber );
```

Comment:
I don't think extern is required
```suggestion
    BaseType_t xApplicationGetRandomNumber( uint32_t * pulNumber );
```